Change position of bad contrast error message

daaAd1 opened the pull request:
#96 
I changed the position of error message in html to top of color settings, but i am not sure if this is what you meant. Please let me know if this is ok.

rugk commented:
Hi @daaAd1,
thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

Please do also note one _important_ precautionary warning of me: In case you did not notice, you have committed (i.e. created commits) with potentially sensitive data like your (full/private) name or mail.
Anyone, who clones your fork/this repo can view these private details, exactly as you've committed them. Also, if this PR gets merged, this will stay in the (commit) history of this project and will thus hard to be removed later.

So if this was unintentional and you want that data to be changed/removed first, better do it now. :smiley: [Here is a good resource](https://stackoverflow.com/a/750182/5008962) ([actually from GitHubs help](https://help.github.com/en/articles/changing-author-info)) on how to change your name/mail afterwards.
Also note that when setting your git [username](https://help.github.com/en/articles/setting-your-username-in-git) and [especially your email](https://help.github.com/en/articles/setting-your-commit-email-address), you can actually write/choose anything there. As for GitHub, it is recommend to use your GitHub username and – if you want to keep your mail private – the GitHub provided `noreply` address. You can [even block pushes of commits with private mails](https://help.github.com/en/articles/blocking-command-line-pushes-that-expose-your-personal-email-address).

So far just the possibilities. I just wanted to serve this as a friendly reminder, so you don't notice this only when it's too late. What you'll do depends solely on you. :smile:

rugk commented on src/options/options.html:
```diff
@@ -148,6 +141,16 @@ <h2 data-i18n="__MSG_subtitleMisc__">Miscellaneous settings</h2>
 						</div>
 					</li>
 
+					<li>
```
Hmm, should not you also add a `.line` css class here? (not tested, I don't know whether it works :smile:)
Or an extra div?

rugk commented on src/options/options.html:
```diff
@@ -148,6 +141,16 @@ <h2 data-i18n="__MSG_subtitleMisc__">Miscellaneous settings</h2>
 					</li>
 
+					<li>
+						<div id="messageContrast" aria-label="info message" data-i18n data-i18n-aria-label="__MSG_ariaMessageInfo__" class="message-box info invisible fade-hide">
```
also better surround this thing with `.message-container` CSS class in case this may later be needed. (currently does not matter AFAIK, but well… :woman_shrugging: :wink: )

Let me know, if this does not work. :smile:

daaAd1 commented:
Hi @rug, thank you for your helpful reply.  I forgot to use noreply email address in this commit :disappointed: I followed the links you provided and hopefully it should be corrected now. 

Tomorrow I will have a look on your comments to code and I will adjust the code.

rugk commented:
Okay fine, but on GitHub your name/mail is not already corrected. Maybe you still need to `git push --force`?

daaAd1 commented:
Hi @rugk , personal info should be corrected, altough I am not sure, where can I see what info was used in this commits. I fixed the position of error message as you requested. I also added message "message-container" class to li wrapper. I also added div with "line" class.

rugk commented on src/options/options.html:
```diff
@@ -147,7 +140,7 @@ <h2 data-i18n="__MSG_subtitleMisc__">Miscellaneous settings</h2>
 						</div>
 					</li>
-
+					
```
The diff on GitHub shows some unused whitespace here. Better remove that.

Did you use any `editorconfig` file/add-on/tool?

daaAd1 replied:
Should be fixed now. 
I wasn't using any editorconfig tool

daaAd1 commented:
Okay mail should be fixed now. Thank you for help with this first issue :).

rugk commented:
I can confirm it's fixed. Merging now… :smile:
